YoCode/UICheck.cs

```diff
@@ -1,21 +1,56 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 
 namespace YoCode
 {
     public class UICheck
     {
-        public static bool UIContainsFeature(String userFileURL, String[] keyWords)
+        public UICheck(List<String> userFilePaths, String[] keyWords)
         {
-            var userFile = File.ReadAllText(userFileURL);
-            var comp = StringComparison.OrdinalIgnoreCase;
+            UIContainsFeature(userFilePaths, keyWords);
+        }
+
+        public UICheck(String userFilePath, String[] keyWords)
+        {
+            UIContainsFeature(userFilePath, keyWords);
+        }
+
+        private void UIContainsFeature(String userFilePath, String[] keyWords)
+        {
+            var userFile = File.ReadAllLines(userFilePath);
+
+            for(var i=0; i<userFile.Length; i++)
+            {
+                // Output code line where this keyword was found.
+                if (ContainsKeyWord(userFile[i], keyWords)) 
+                {
+
+                    ListOfMatches.Add(i + 1);
```

Comment:
This seems a little odd, the filename is being passed into this method but the line numbers are being collected into a property without reference to the file name. I may be missing something, but it looks like you won't be able to tell which file the line-matches came from.

Unless we don't care which file they come from, and we only want to know if any of the files looked like that have some UI updates? If so, then you could maybe replace `List<int> ListOfMatches` with a `bool`? (that might be a :poop: option, so make sure you think about it what you want to do with these line numbers before doing something like that 😄)

Reply:
Will add TODO comment on this line. Implementation  is not finished of this feature as we only need to get yes/no answer from above method for sprint0 (so that output on the console would look something like: Evidence in UI: Yes/No).